tests/unit/test_parameters/test_parameter_values.py

```diff
@@ -159,6 +159,20 @@ def test_check_parameter_values(self):
                 {"Negative electrode LAM constant propotional term": 1}
             )
 
+    def test_parameter_deprecation(self):
```

Comment:
This test should test that `parameter_values.update({"1 + dlnf/dlnc": 1})` errors. Currently it's not testing anything useful. Also, use `assertRaisesRegex`